### PR TITLE
Initial Release - AWS RI Purchase Recommendations

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

Initial Release - AWS RI Purchase Recommendations

### Issues Resolved

N/A

### Contribution Check List

- [X] All tests pass.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/reserved_instances/recommendations/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## AWS Reserved Instances Recommendation Policy Template
+
+## NOTE: These RI Purchase Recommendations are generated by AWS. This policy is will raise incidents based on those recommendations.
```
Comment (srpomeroy) [inline cost/aws/reserved_instances/recommendations/README.md]: Should be "this policy will raise..."

File: cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+auth "aws", type: "aws" do
+  version "4"
+  service "ce"
+  region "us-east-1"
+  access_key cred("AWS_ACCESS_KEY_ID")
+  secret_key cred("AWS_SECRET_ACCESS_KEY")
+end
+
+#pagination "aws_pagination" do
```
Comment (srpomeroy) [inline cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt]: What's up with the pagination block that's commented out?

Comment (dfrankel33) [reply cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt]: this pagination method wasn't supported when initially developed. but it is now! thanks for pointing it out, i'll update

File: cost/aws/reserved_instances/recommendations/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+This policy has the following input parameters required when launching the policy.
+
+- *Look Back Period* - Specify the number of days of past usage to analyze.
+- *Service* - Specify the AWS Service for which to search for RIs
+- *EC2 Specification* - If the Service is "EC2", specify whether to look for Standard or Convertible RIs.  For Services besides EC2, this parameter will be ignored.
+- *RI Term* - Specify he Term length for the RI.
+- *Payment Option* - Optionally, specify the payment option for this RI.
+- *Net Savings Threshold* - Specify the minimum Net Savings that should result in an RI purchase recommendation
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
```
Comment (cdwilhelm) [inline cost/aws/reserved_instances/recommendations/README.md]: should match label as suggested below

**Pull request merged**